docs/api.md

```diff
@@ -343,7 +384,9 @@ Advanced parameters (optional):
 
 ### Examples
 
-#### Request
+#### Chat Request (Streaming)
```

Comment:
I think we might have left out the contents of this section

Reply:
I didn't really have anything to add, but the levels are there for consistency with the rest of the doc